api/docs/CMake_rundoxygen.cmake

```diff
@@ -256,6 +256,11 @@ if (embeddable)
   endforeach ()
```

Comment:
> previously written to page_user_docs.js,

nit: It was written to a file per page: not just to this one particular page of ours.